cmd/oras/manifest/push.go

```diff
@@ -135,16 +148,26 @@ func pushManifest(opts pushOptions) error {
 		}
 	}
 
+	tagBytesNOpts := oras.DefaultTagBytesNOptions
+	tagBytesNOpts.Concurrency = opts.concurrency
+
 	// outputs manifest's descriptor
 	if opts.OutputDescriptor {
+		if len(opts.extraRefs) != 0 {
+			oras.TagBytesN(ctx, manifests, mediaType, contentBytes, opts.extraRefs, tagBytesNOpts)
+		}
 		descJSON, err := opts.Marshal(desc)
 		if err != nil {
 			return err
 		}
 		return opts.Output(os.Stdout, descJSON)
+	} else {
+		display.Print("Pushed", opts.targetRef)
+		if len(opts.extraRefs) != 0 {
+			oras.TagBytesN(ctx, &display.TagManifestStatusPrinter{Repository: repo}, mediaType, contentBytes, opts.extraRefs, tagBytesNOpts)
+		}
 	}
```

Comment:
`else` is not needed since the previous `if` block ends with `return`.

```suggestion
	}
	
	fmt.Println("Pushed", opts.targetRef)
	if len(opts.extraRefs) != 0 {
		oras.TagBytesN(ctx, &display.TagManifestStatusPrinter{Repository: repo}, mediaType, contentBytes, opts.extraRefs, tagBytesNOpts)
	}
```

Reply:
done